conf/are.conf.part

```diff
@@ -4,7 +4,7 @@
 
 source src_ch_are_landschaftstypen : def_queryable_features
 {
-    sql_db = are
+    sql_db = are_dev
```

Comment:
The host wont change anymore, always pg.bgdi.ch.
When you deploy the sphinx config to prod, only the databases suffixes will be replaced with _prod
the db suffixes are configured here: 
https://github.com/geoadmin/service-sphinxsearch/blob/11c4534e1627978d94bdcaa3f0136554a2c3a4f1/deploy/deploy-conf-only.cfg#L18

and replaced here:
https://github.com/geoadmin/service-sphinxsearch/blob/11c4534e1627978d94bdcaa3f0136554a2c3a4f1/deploy/hooks/post-restore-code#L69